### ISSUE TITLE
Logs break on binary data

### ISSUE THREAD

**Issue by nullproxy** [kusanagi/katana-sdk-php7#53]
When a value that contains content such as *binary* is provided as the output for a log, these are no longer visible in `stdout`.

Additionally, after that log, all other logs are affected, with no further output to `stdout`. This was used in a **Middleware** processing requests, and not even on subsequent requests were logs visible.

This was using version **1.1.5** of the framework, and version **1.1.8** of the SDK, with system info below:

```
Linux work 4.2.0-42-generic #49~14.04.1-Ubuntu SMP Wed Jun 29 20:22:11 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by fcastilloes** [comment]
Have you tried without the framework?

Here is the output for a test I just did with the **1.1.10** version of the SDK, with no relevant changes about logging since the **1.1.8**. For this I generated a random binary file like this:

```
dd if=/dev/urandom of=binary bs=1 count=1024
```

Then I added a log in the startup of an action:

```php
$service->startup(function(Service $service) {
    $service->log("Setting things up");
    $service->log(file_get_contents(__DIR__ . '/binary'));
```

After launching the SDK alone I get this:

![image](https://user-images.githubusercontent.com/2611437/29936796-c5c3206a-8e83-11e7-8beb-5d836653698d.png)

While using the framework I get this:

![image](https://user-images.githubusercontent.com/2611437/29937040-861c320c-8e84-11e7-9bbe-2fae0924452c.png)

Note there is no startup logs from the SDK, but there are logs for the shutdown. I suppose the framework can't handle the binary and skips the whole log stack from the SDK. In this case the startup logs.

I can't see a bug in the SDK here.
